README.md

```diff
@@ -21,34 +21,25 @@ npm run start:dev
 
 Navigate to `http://localhost:3000` and login to the application using HMPPS Auth dev credentials e.g. `AUTH_ADM/password123456`
 
-### Test
+### Unit Test
 
 `npm run test`
 
 ### Lint
 
 `npm run lint`
- 
-## Dependencies
-
-- hmpps-auth - for authentication
-- redis - session store and token caching
-
 
-## Integration tests
+### Integration Test
 
-For local running, start a test db, redis, and wiremock instance by:
+The integration tests require a different docker-compose stack and a different application configuration. Run each of the following commands in its own shell. 
 
 `docker-compose -f docker-compose-test.yml up`
 
-Then run the server in test mode by:
+`npm start:test`
```

Comment:
@Gweaton realised that this should be `npm run start:test`

Reply:
done